Automattic/wp-calypso
# user-settings: reduxify `TwoStep` and its children

This PR is based on #49000 and will merge into it after both got reviewed!

#### Changes proposed in this Pull Request

* Remove `userSettings` prop from `TwoStep` and its children

#### Testing instructions

* Go to `/me/security/two-step`
* Go through the setup flow and make sure it's behaving exactly like on prod
* Disable two factor authentication and make sure it's behaving exactly like on prod

part of #24162
